.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,8 +10,6 @@ Request for comments document (if applies):
 
 ### Checklist ✅
 
-- [ ] The code architecture and patterns are consistent with the rest of the codebase.
-- [ ] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
-- [ ] The PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, whenever it should be included in the “Added”, “Fixed” or “Changed” section of the CHANGELOG. Note: when included in the CHANGELOG, the title of the PR will be used as entry, please make sure it is clear and suitable.
```

Comment:
Users can't set the label, so I have simplified this line

Reply:
It could be useful for the the contributors / maintainers as a checklist before merging?

Reply:
Makes sense, I'll just make clear that it's for the contributors / maintainers 👍